homeassistant/util/pressure.py

```diff
@@ -0,0 +1,48 @@
+"""Distance util functions."""
```

Comment:
This should probably be part of `util/unit_system.py`

Reply:
Looks like I forgot a place to rename to pressure instead of distance.  I will fix that.  

I did consider adding to `util/unit_system.py`, but I stopped since it requires a default unit for pressure.  What should the default unit be?  inHg is a common unit for atmospheric pressure.  in/ftH20 are common units for piping pressure.  psi is also a common unit for piping pressure.  For the weather platform, it would make sense to use hPa and inHg, but for someone with a homebrew setup that wants to monitor the pressure of a keg, it might be kPa and psi.

Reply:
Can there be defaults for large categories like `pressure_atmospheric` and `pressure_internal`?  This might solve 50-75% of the problem here.